packages/php-wasm/compile/build.js

```diff
@@ -96,13 +96,13 @@ const platformDefaults = {
 		PHP_VERSION: '8.0.24',
 		WITH_LIBZIP: 'yes',
 		WITH_SQLITE: 'yes',
+		WITH_LIBXML: 'yes',
+		WITH_MBSTRING: 'yes',
```

Comment:
Building with libxml adds 1MB to the bundle size, but I realize it's critical for this PR to be useful 🤔 

I need to think a bit more about this. Emscripten seems to support [runtime dynamic linking](https://emscripten.org/docs/compiling/Dynamic-Linking.html#runtime-dynamic-linking-with-dlopen), but it's a small project in itself I don't want to block this PR on it. Maybe an additional MB wouldn't be that bad for now.